src/sentry/tasks/groupowner.py

```diff
@@ -23,7 +23,7 @@
     max_retries=5,
 )
 def process_suspect_commits(event_id, event_platform, event_frames, group_id, project_id, **kwargs):
-    metrics.incr("sentry.tasks.process_suspect_commits.start")
+    metrics.incr("sentry.tasks.process_suspect_commits.start", tags={"project": project_id})
```

Comment:
Let's not add project IDs as tags to `metrics.*` calls, please. It will open doors to high-cardinality metrics that make our metrics provider not very happy.